src/abstractdataframe/selection.jl

```diff
@@ -31,6 +22,8 @@ struct ByRow{T}
     fun::T
 end
 
+Base.broadcastable(x::ByRow) = Ref(x)
```

Comment:
Is this tested?

Reply:
yes, this is tested in https://github.com/JuliaData/DataFrames.jl/pull/2155/files#diff-002e65995072b19e4150f93385cb5f61R897

Reply:
According to Coverall it isn't. Is it wrong?
https://coveralls.io/builds/29767300/source?filename=src%2Fabstractdataframe%2Fselection.jl#L25

Reply:
Strange - I added an explicit test already (I now double checked and the test is `transform(df, names(df) .=> ByRow(-))`). I have added 2 more tests of this.